docs/04-operation-guides/security/sec-06-access-expose-kiali-grafana.md

```diff
@@ -50,21 +50,27 @@ By default, Kyma does not expose Kiali, Grafana, and Jaeger. However, you can st
   </details>
 </div>
 
->**NOTE:** kubectl port-forward does not return. You will have to cancel it with Ctrl+C if you want to stop port forwarding.
+>**NOTE:** kubectl port-forward does not return. To stop port forwarding, cancel it with Ctrl+C.
 
 2. To access the respective service's UI, open `http://localhost:20001` (for Kiali), `http://localhost:3000` (for Grafana), or `http://localhost:16686` (for Jaeger) in your browser.
 
 ## Expose Kiali, Grafana, and Jaeger securely
 
-Kyma manages an [OAuth2 Proxy](https://oauth2-proxy.github.io/oauth2-proxy/) instance to secure access to Kiali, Grafana and Jaeger. To make the services accessible, configure OAuth2 Proxy by creating a Kubernetes `Secret` with your identity provider credentials.
+Kyma manages an [OAuth2 Proxy](https://oauth2-proxy.github.io/oauth2-proxy/) instance to secure access to Kiali, Grafana, and Jaeger. To make the services accessible, configure OAuth2 Proxy by creating a Kubernetes `Secret` with your identity provider credentials.
```

Comment:
```suggestion
Kyma manages an [OAuth2 Proxy](https://oauth2-proxy.github.io/oauth2-proxy/) instance to secure access to Kiali, Grafana, and Jaeger. To make the services accessible, configure OAuth2 Proxy by creating a Kubernetes Secret with your identity provider credentials.
```
Is there a reason why `Secret` is formatted as code here? It doesn't really fit into any of the use described in our [guidelines](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/03-formatting.md#use-code-font-for-these-items) for me, and as far as I recall, and from what I see, we don't normally format it in our docs ([example](https://kyma-project.io/docs/kyma/latest/03-tutorials/00-serverless/svls-09-log-into-private-packages-registry/#create-a-secret)).

Reply:
Not really, thank you for spotting it!